jbmc/src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -100,7 +100,7 @@ void java_bytecode_languaget::get_language_options(const cmdlinet &cmd)
     extra_entry_points.end(),
     std::back_inserter(extra_methods),
     [](const std::string &pattern) {
-      return util_make_unique<load_method_by_regext>(pattern);
+      return build_load_method_by_regex(pattern);
```

Comment:
Could you put `build_load_method_by_regex` as an argument of `std::transform` directly?